From b4ca801960ca311017816da5b16aa77afbb7e783 Mon Sep 17 00:00:00 2001
From: Alan Modra <amodra@gmail.com>
Date: Mon, 8 Nov 2021 08:19:34 +1030
Subject: [PATCH 72/97] Correct ld script wildcard matching description

Goes with commit 68bbb9f788d0

	* ld.texi (Input Section Wildcards): Delete paragraph incorrectly
	saying '*' does not match '/'.
---
 ld/ld.texi | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/ld/ld.texi b/ld/ld.texi
index db410f0bf2d..e6c1f42c985 100644
--- a/ld/ld.texi
+++ b/ld/ld.texi
@@ -5022,13 +5022,6 @@ character may be used to specify a range of characters, as in
 quotes the following character
 @end table
 
-When a file name is matched with a wildcard, the wildcard characters
-will not match a @samp{/} character (used to separate directory names on
-Unix).  A pattern consisting of a single @samp{*} character is an
-exception; it will always match any file name, whether it contains a
-@samp{/} or not.  In a section name, the wildcard characters will match
-a @samp{/} character.
-
 File name wildcard patterns only match files which are explicitly
 specified on the command line or in an @code{INPUT} command.  The linker
 does not search directories to expand wildcards.
-- 
2.34.0

